apps/user-office-frontend/src/components/internalReview/InternalReviewsTable.tsx

```diff
@@ -64,14 +64,17 @@ const InternalReviewsTable = ({
     />
   );
 
-  const deleteInternalReview = async (id: string | number) => {
+  const deleteInternalReview = async (

```

Comment:
I think the changes here are not needed now after cache is cleared.

Reply:
Okay will do that was just checking where it was coming from.